### PR TITLE
DOCAPI-7430: MergeTree INDEX bloom filter docs.

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Documentation for #5678.

### CODE REVIEW

File: docs/en/operations/table_engines/mergetree.md [hunk 1]
```diff
@@ -302,6 +305,12 @@ SELECT count() FROM table WHERE u64 * i32 == 10 AND u64 * length(s) >= 1234
 
     The same as `ngrambf_v1`, but instead of ngrams stores tokens, which are sequences separated by non-alphanumeric characters.
 
+- `bloom_filter` — Stores [bloom filter](https://en.wikipedia.org/wiki/Bloom_filter) for the specified columns.
```
Comment (zhang2014) [inline docs/en/operations/table_engines/mergetree.md]: It actually accepts an optional parameter: [false positives](http://pages.cs.wisc.edu/~cao/papers/summary-cache/node8.html)

Comment (BayoNet) [reply docs/en/operations/table_engines/mergetree.md]: @zhang2014 Does this optional parameter is a probability of false positive response?

Comment (zhang2014) [reply docs/en/operations/table_engines/mergetree.md]: Yes, it should be a float between 0 and 1.

File: docs/en/operations/table_engines/mergetree.md [hunk 2]
```diff
@@ -45,7 +45,10 @@ CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
 [SETTINGS name=value, ...]
 ```
 
-For a description of request parameters, see [request description](../../query_language/create.md).
+For a description of parameters, see the [CREATE query description](../../query_language/create.md).
+
+!!!note "Note"
```
Comment (blinkov) [inline docs/en/operations/table_engines/mergetree.md]: * lost space after !!! ?
* needs link/mention of feature flag to enable it

**Review by blinkov — CHANGES_REQUESTED** [review]
and some merge conflicts

File: docs/en/operations/table_engines/mergetree.md (hunk 2 above)
Comment (BayoNet) [reply docs/en/operations/table_engines/mergetree.md]: There is the [Data Skipping Indexes](https://clickhouse.yandex/docs/en/operations/table_engines/mergetree/#data-skipping-indices-experimental) section. I think its enough.

**Pull request merged**